### PR TITLE
Switch to PG10 packages in our install scripts

### LINKED ISSUE #103: Update install scripts to detect PostgreSQL 10
PostgreSQL 10 will be the "favored" version of PostgreSQL as of Citus 7.1, so our scripts should assume users want it. The eight `deb`/`rpm` scripts in the `gh-pages` branch need to be updated (search for any occurrences of `9.6` or `96` and update appropriately).

### PR DESCRIPTION
**Pull request by byucesoy**
Fixes #103

### CODE REVIEW

**Comment by jasonmp85** [comment]
@sumedhpathak ready for a quick look on that second commit. I'm testing on each OS using ``docker run -v`pwd`/community:/community -it --rm <OS> /community/rpm.sh``, i.e. mounting the community directory within a docker container and then running the script. I've done CentOS 6/7, Fedora 25/26 and Oracle Linux 6/7 now. Everything looks good (I haven't tested Red Hat Enterprise Linux, since I think I'd have to spin up an EC2 instance for that).

**Comment by jasonmp85** [comment]
I was wrong about Fedora. I tried 25 and 26; 27 just came out yesterday and doesn't work, of course.

File: enterprise/rpm.sh [hunk 1]
```diff
@@ -186,26 +186,16 @@ detect_repo_url ()
...
-      pkg_os='ami201503-'
-      pkg_version='2'
-      ;;
     ol)
       pkg_os='oraclelinux'
       ;;
     fedora)
       family='fedora'
       family_short='fedora'
+      pkg_version='3'
```
Comment (sumedhpathak) [inline enterprise/rpm.sh:198]: Fedora still seems to be at package version 2. Is this not yet updated on their website?

Comment (jasonmp85) [reply enterprise/rpm.sh:198]: Yeah, and many of the others still say `1` on their website.

**Pull request merged**